### PR TITLE
Loading state

### PR DESCRIPTION
**Pull request by tobygabriella**
created a loading custom hook, component, and css file for the loading state

### CODE REVIEW

File: Frontend/src/Components/CustomHooks/useLoading.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+const useLoading = () => {
```
Comment (conniewu3) [inline Frontend/src/Components/CustomHooks/useLoading.jsx:3]: love the use of a hook here!

File: Frontend/src/Components/OtherUsersProfile/OtherUsersProfile.jsx [hunk 2]
```diff
@@ -33,18 +43,18 @@ const OtherUsersProfile = () => {
       } catch (err) {
         setError(err.message);
```
Comment (conniewu3) [inline Frontend/src/Components/OtherUsersProfile/OtherUsersProfile.jsx:44]: should this be setErrorState?

**Review by conniewu3 — APPROVED** [review]
looks good!

File: Frontend/src/Components/OtherUsersProfile/OtherUsersProfile.jsx (hunk 2 above)
Comment (tobygabriella) [reply Frontend/src/Components/OtherUsersProfile/OtherUsersProfile.jsx:44]: yes! made the changes

**Pull request merged**